lms/djangoapps/courseware/tests/test_split_module.py

```diff
@@ -167,65 +168,79 @@ class TestVertSplitTestVert(SplitTestBase):
     def setUp(self):
         # We define problem compenents that we need but don't explicitly call elsewhere.
         # pylint: disable=unused-variable
-        super(TestVertSplitTestVert, self).setUp()
+        super(TestSplitTestVertVert, self).setUp()
 
-        # vert <- split_test
-        # split_test cond 0 = vert <- {video, problem}
-        # split_test cond 1 = vert <- {video, html}
-        vert1 = ItemFactory.create(
-            parent_location=self.sequential.location,
-            category="vertical",
-            display_name="Split test vertical",
-        )
         c0_url = self.course.id.make_usage_key("vertical", "split_test_cond0")
         c1_url = self.course.id.make_usage_key("vertical", "split_test_cond1")
 
         split_test = ItemFactory.create(
-            parent_location=vert1.location,
+            parent_location=self.sequential.location,
             category="split_test",
             display_name="Split test",
             user_partition_id='0',
             group_id_to_child={"0": c0_url, "1": c1_url},
         )
 
-        cond0vert = ItemFactory.create(
+        # vert <- split_test
+        # split_test cond 0 = vert <- {video, problem}
+        # split_test cond 1 = vert <- {video, html}
+        vert0 = ItemFactory.create(
             parent_location=split_test.location,
             category="vertical",
             display_name="Condition 0 vertical",
             location=c0_url,
         )
-        video0 = self._video(cond0vert, 0)
-        problem0 = self._problem(cond0vert, 0)
 
-        cond1vert = ItemFactory.create(
+        vert1 = ItemFactory.create(
             parent_location=split_test.location,
             category="vertical",
             display_name="Condition 1 vertical",
             location=c1_url,
         )
+
+        cond0vert = ItemFactory.create(
+            parent_location=vert0.location,
+            category="vertical",
+            display_name="Condition 0 vertical child",
+        )
```

Comment:
Why was it necessary to create new vertical children within the "conditional" verticals?